### ISSUE TITLE
Issue after updating to 1.0.3

### ISSUE THREAD

**Issue by jonpoeschek** [Asd-g/AviSynthPlus-RIFE#4]
In my AviSynth project, after ~ 105812 frames RIFE is returning every other frame with a warping of what looks to be frame 105812 + the next two frames. I've attached a video of what it looks like in AvsPmod and when it's rendered.

The issues is not present if I downgrade back to RIFE 1.0.2.

The AviSynth Script is as follows:

```
SetFilterMTMode("DEFAULT_MT_MODE", MT_MULTI_INSTANCE)
Threads=16

LSMASHVideoSource("_encode-3-TFM.mp4")
ConvertBits(32).ConvertToPlanarRGB(matrix="Rec601").RIFE(model=22, fps_num=24000, fps_den=1001, gpu_thread=8).ConvertBits(8).ConvertToYV12(matrix="Rec601")

Prefetch(Threads)
```

This is the media info for the file:
[_encode-3-TFM.mp4.txt](https://github.com/Asd-g/AviSynthPlus-RIFE/files/11988820/_encode-3-TFM.mp4.txt)

https://github.com/Asd-g/AviSynthPlus-RIFE/assets/19938440/394e08ce-b876-4c5b-a517-9f3da839a852

**Comment by Asd-g** [comment]
Can you share a sample that I can use to reproduce the issue?

Did you try with default gpu_thread?

**Comment by jonpoeschek** [comment]
Yes it happens if I don't specify gpu threads.

I've only seen this happen with files that have more than ~100000 frames (basically any 1.5 hour long episode I've been working on)

I encoded a non-lossless mp4 and verified the same behavior. You can access it here:
https://1drv.ms/f/s!AvMHKRqgtwEd1C8w50MS3k-AL68b?e=y7lHz6

**Comment by Asd-g** [comment]
Thanks for the sample. Try the new version.

**Comment by jonpoeschek** [comment]
Works great! Thanks for your help getting that fixed :)

**Comment by jonpoeschek** [comment]
Closing issue as this is now resolved with 1.0.4
